.dockerignore

```diff
@@ -0,0 +1 @@
+storage/
```

Comment:
you can also add `.git`

Reply:
u dont need to add .git.. git ignores .git by default.. :o